data/yam/agama/auto/default_sle.json

```diff
@@ -12,5 +12,14 @@
   "root": {
     "password": "$6$vYbbuJ9WMriFxGHY$gQ7shLw9ZBsRcPgo6/8KmfDvQ/lCqxW8/WnMoLCoWGdHO6Touush1nhegYfdBbXRpsQuy/FTZZeg7gQL50IbA/",
     "hashedPassword": true
+  },
+  "scripts": {
+    "post": [
+      {
+        "name": "enable sshd",
+        "chroot": true,
+        "body": "#!/bin/bash \n echo 'PermitRootLogin yes' > /etc/ssh/sshd_config.d/root.conf \n firewall-cmd --permanent --add-service=ssh \n firewall-cmd --reload \n systemctl enable sshd.service \n"
```

Comment:
Thanks for your new PR, it will solve many problems 
Based on my manual test, `echo 'PermitRootLogin yes' > /etc/ssh/sshd_config.d/root.conf` could be file :)

Reply:
Instead, `firewall-cmd` is not installed: can be removed from script.